### PR TITLE
StartScopeSpan out param

### PR DESCRIPTION
**Pull request by sandeshp**
https://github.com/census-instrumentation/opencensus-csharp/issues/45 - StartScopedSpan out param

### CODE REVIEW

**Comment by sandeshp** [comment]
I signed it!

File: src/Samples/TestZipkin.cs [hunk 1]
```diff
@@ -55,14 +55,13 @@ private static void DoWork(int i)
             // 6. Get the global singleton Tracer object
             ITracer tracer = Tracing.Tracer;
 
+            ISpan span = null;
             // 7. Start another span. If another span was already started, it'll use that span as the parent span.
             // In this example, the main method already started a span, so that'll be the parent span, and this will be
             // a child span.
-            using (OpenCensus.Common.IScope scope = tracer.SpanBuilder("DoWork").StartScopedSpan())
+            using (OpenCensus.Common.IScope scope = tracer.SpanBuilder("DoWork").StartScopedSpan(out span))
```
Comment (SergeyKanzhelev) [inline src/Samples/TestZipkin.cs]: you can define `span` right inside the `out`. It is a new syntax in C#. Something like this looks much better:

``` csharp
using (var scope = tracer.SpanBuilder("DoWork").StartScopedSpan(out var span))
```

Comment (SergeyKanzhelev) [reply src/Samples/TestZipkin.cs]: so you wouldn't need globally scoped `ISpan span = null;`

Comment (sandeshp) [reply src/Samples/TestZipkin.cs]: Done, Thanks !

**Pull request merged**